game/theater/iadsnetwork/iadsnetwork.py

```diff
@@ -120,6 +120,11 @@ def skynet_nodes(self, game: Game) -> list[SkynetNode]:
                 # Skip culled ground objects
                 continue
 
+            if node.group.alive_units == 0:
```

Comment:
That will do the trick. Just one thing is missing: this will also prevent dead statics from being spawned. So if `group.units[0].is_static` is true it should not be skipped. Thats a limitation of DCS that we can sawn statics as dead but vehicles have to be "killed" and can not be spawned as dead directly.
https://github.com/dcs-liberation/dcs_liberation/blob/cc4237d0762cea396fe56134adf2bc64fa1682c4/game/theater/iadsnetwork/iadsnetwork.py#L38-L58

Reply:
Also this will not check if the group of the connection has alive units. But atm connections can only be Comms and Power and we only have statics for them so this will not raise an exception. As this is a temporary hotfix thats not a big deal so just a side note to keep in mind with the refactoring done with the PRs :)